lib/house.rb

```diff
@@ -0,0 +1,33 @@
+class House
+  attr_reader :price,
+              :address,
+              :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    false
+  end
```

Comment:
This should be a conditional based on the price of the house.  This passes the test you have below because you are only checking against a house that is below market average.